app/models/ems_refresh/save_inventory.rb

```diff
@@ -19,12 +25,36 @@ def save_ems_inventory(ems, hashes, target = nil, disconnect = true)
 
     # Handle updates to the ext_management_system
     update_attributes!(ems, hashes[:ems], [:type]) unless hashes[:ems].nil?
+
+    _log.info "#{log_header} Saving inventory...Complete"
   end
 
   def save_ems_inventory_no_disconnect(ems, hashes, target = nil)
     save_ems_inventory(ems, hashes, target, false)
   end
 
+  def queue_save_ems_inventory(ems, hashes, target = nil)
+    args = [[ems.class.name, ems.id]]
+    args << [target.class.name, target.id] unless target.nil?
+
+    queue_opts = {
+      :queue_name  => 'inventory',
+      :class_name  => name,
+      :method_name => 'save_ems_inventory_from_queue',
+      :role        => 'ems_inventory',
+      :zone        => ems.my_zone,
+      :args        => args,
+      :data        => hashes,
```

Comment:
> we have a filter on :data to only print a limited amount of data. If I was able to do the same for :args. Do you think this would work for your use case?

Yes that would work